ui/load/load.go

```diff
@@ -86,6 +85,8 @@ type Load struct {
 	SubscribeKeyEvent   func(eventChan chan *key.Event, pageID string) // Widgets call this function to recieve key events.
 	UnsubscribeKeyEvent func(pageID string) error
 	ReloadApp           func()
+	DesktopNotifier     func(notifier interface{})
+	UpdateBalance       func()
```

Comment:
Comments please. I recognise this hasn't been the custom, but it'd help a code reader understand why these properties exist without having to dig to find where and how they're used.

Reply:
Update: I'm sure we can do without them even.